plume-models/src/schema.rs

```diff
@@ -47,160 +47,160 @@ table! {
     }
 }
 
+table! {
+    comment_seers (id) {
+        id -> Nullable<Integer>,
```

Comment:
this is not supposed to be Nullable, but I don't get where it come from, your migrations aren't touching this table

Reply:
Diesel is regenerating the schema when running migrations, so it changed when I tested with SQlite.

Reply:
For some reason, comment_seers's id is an `INTEGER PRIMARY KEY AUTOINCREMENT`, not an `INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT` on sqlite (but fine on pg I assume), that's why the scheme was invalid